lib/dogma/rules/module_doc.ex

```diff
@@ -43,11 +43,12 @@ defmodule Dogma.Rules.ModuleDoc do
     end
   end
 
-  defp check_node({:defmodule, m, [_, [do: module_body]]} = node, errors) do
+  defp check_node({:defmodule, m, [mod, [do: module_body]]} = node, errors) do
+    {_,_,names} = mod
     if module_body |> moduledoc? do
       {node, errors}
     else
-      {node, [error( m[:line] ) | errors]}
+      {node, [error( m[:line], Enum.join(names, ".") ) | errors]}
```

Comment:
Ah, good point. One thing is that
```
iex(1) [:ModName] |> Module.concat |> to_string
"Elixir.ModName"
```

That 'Elixir' prefix in the errror message seems a little noisy but if it doesn't bother you I'll change it.